### PR TITLE
Update vNext to use .Net Tooling RC3 features, compatibility with VS2017 RC3

### PR DESCRIPTION
**Pull request by attilah**
Update vNext to use .Net Tooling RC3 features, compatibility with VS2017 RC3

### CODE REVIEW

File: vNext/src/NuGet/Microsoft.Orleans.OrleansSQLUtils.nuspec [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <metadata>
     <id>Microsoft.Orleans.OrleansSqlUtils</id>
     <version>$version$</version>
-    <title>icrosoft Orleans Sql Utilities</title>
+    <title>Microsoft Orleans Sql Utilities</title>
```
Comment (ReubenBond) [inline vNext/src/NuGet/Microsoft.Orleans.OrleansSQLUtils.nuspec]: (nit) SQL should be capitalized, too.

File: vNext/src/Directory.Build.props [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<Project>
+  <PropertyGroup Label="NuGet">
+    <IsPackable>false</IsPackable>
```
Comment (ReubenBond) [inline vNext/src/Directory.Build.props]: What does this do?

File: vNext/src/OrleansCodeGeneratorBuildMetaPackage/OrleansCodeGeneratorBuildMetaPackage.csproj [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    <TargetFramework>net462</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="..\Orleans.SDK.targets">
+      <Pack>true</Pack>
+      <PackagePath>build\Microsoft.Orleans.OrleansCodeGenerator.Build.targets</PackagePath>
+      <Visible>false</Visible>
+    </None>
+    <None Include="$(OutDir)/ClientGenerator.exe*">
```
Comment (ReubenBond) [inline vNext/src/OrleansCodeGeneratorBuildMetaPackage/OrleansCodeGeneratorBuildMetaPackage.csproj]: Would globbing work for these?

**Comment by ReubenBond** [comment]
LGTM, just a couple of questions. I  think it's fine to commit if we can verify that the resulting packages do indeed work as intended.

**Comment by galvesribeiro** [comment]
LGTM. Thanks @attilah

File: vNext/src/NuGet/Microsoft.Orleans.OrleansSQLUtils.nuspec (hunk 1 above)
Comment (attilah) [inline vNext/src/NuGet/Microsoft.Orleans.OrleansSQLUtils.nuspec]: Look at this in src ;-) but these nuspecs will be removed in vNext, so don't have to bother with it, just "accidentally" fixed this typo.

File: vNext/src/Directory.Build.props (hunk 2 above)
Comment (attilah) [inline vNext/src/Directory.Build.props]: It excludes all csprojs from below from the "Pack" target by default.

File: vNext/src/OrleansCodeGeneratorBuildMetaPackage/OrleansCodeGeneratorBuildMetaPackage.csproj (hunk 3 above)
Comment (attilah) [inline vNext/src/OrleansCodeGeneratorBuildMetaPackage/OrleansCodeGeneratorBuildMetaPackage.csproj]: I am using globbing. The reason for having multiple entries for ClientGenerator this is to not to copy the build generated json files. Later I'd like to address of the "flat output directory" that we are having now.

**Comment by attilah** [comment]
@ReubenBond I've verified the generated nupkgs against the current ones we've in vNext master.

**Comment by galvesribeiro** [comment]
:shipit:

**Pull request merged**